src/rgw/rgw_cr_rados.h

```diff
@@ -11,6 +11,9 @@
 #include "rgw_sal_rados.h"
 #include "common/WorkQueue.h"
 #include "common/Throttle.h"
+#include "rgw_cr_rest.h"
+#include "rgw_rest_conn.h"
+#include "rgw_rados.h"
```

Comment:
`rgw_cr_rados.h` shouldn't need these includes; you can use forward declarations instead, and move the includes into `rgw_cr_rados.cc`